packages/mobile/src/components/details-tile/DetailsTile.tsx

```diff
@@ -107,6 +117,37 @@ const useStyles = makeStyles(({ palette, spacing, typography }) => ({
     marginBottom: spacing(6)
   },
 
+  progressInfoSection: {
+    gap: spacing(2),
+    width: '100%',
+    marginBottom: spacing(4),
+    paddingHorizontal: spacing(4)
+  },
+
+  progressInfoContainer: {
+    flexDirection: 'row',
+    justifyContent: 'space-between'
+  },
+
+  progressTextContainer: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    alignContent: 'center',
+    gap: 2
+  },
+
+  progressInfoText: {
+    fontSize: typography.body1.fontSize,
+    color: palette.neutralLight4
+  },
+
+  progressTextSlider: {
```

Comment:
i think you can remove some of the styles here now that ProgressInfo is its own component?